DEVELOPER_GUIDE.md

```diff
@@ -17,6 +17,16 @@ $ nox -rs format
 
 # Run the test suite
 $ nox -rs test
+
+```
+
+## Install and Run Docker Image
+Note that integration tests require docker to be installed and running, and downloads quite a bit of data from over the internet and hence take few minutes to complete.
```

Comment:
Nit: I know you copy-pasted this from below, but let's keep it functional. I would just say "Integration tests require [docker](link ...)." The whole "it takes a few minutes ... " seems unnecessary.

Does one have to run the image before running tests? If not, say "Integration tests will auto-start the docker image. To start it manually:" or something like that.

Reply:
Thank you for your feedback. Done!